spec/dishes_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menu'
+
+describe Menu do
+  
+  subject(:menu) { described_class.new }  
+    
+  describe '#list_dishes' do
```

Comment:
This is a good starting unit test but could maybe be taken out as further tests implicitly check this anyway